src/badge.jsx

```diff
@@ -1,9 +1,54 @@
 import React from 'react';
 import Typography from './styles/typography';
 import getMuiTheme from './styles/getMuiTheme';
-import StylePropable from './mixins/style-propable';
 
-// Badge
+function getStyles(props, state) {
+  const {
+    primary,
+    secondary,
+  } = props;
+
+  const {
+    badge,
+  } = state.muiTheme;
+
+  const [
```

Comment:
Perfectly readable and in less lines of code. I think it's just what you're used to. To me this is the perfect use case for nested ternaries. (But what do I know?! :laughing: )